rrmngmnt/resource.py

```diff
@@ -14,23 +13,14 @@ def warn(self, *args, **kwargs):
             """
             self.warning(*args, **kwargs)
 
-    class ProgressHandler(logging.StreamHandler):
-        """
-        Handle progress bar output and show it under sys.stdout
-        """
-        def emit(self, record):
-            show_progress_bar = hasattr(record, 'show_progress_bar')
-            if show_progress_bar:
-                super(Resource.ProgressHandler, self).emit(record)
-
     def __init__(self):
         super(Resource, self).__init__()
         logger = logging.getLogger(self.__class__.__name__)
-        ph = self.ProgressHandler(sys.stdout)
-        ph.setLevel(logging.DEBUG)
-        logger.addHandler(ph)
         self._logger_adapter = self.LoggerAdapter(logger, {'self': self})
 
+    def logger_handler(self, status):
```

Comment:
I still think this method doesn't belong to Resource class.
what does it mean ? "logger_handler"  ... in the context of Resource class ...

it is not general and doesn't belong to this general base class ...